main.rb

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env ruby
+
+require 'pry'
+require './books'
+require './classroom'
+require './corrector'
+require './person'
+require './rental'
+require './students'
+require './teacher'
+
+NEW_LINE = "\n".freeze
+
+$person_list = []
+$book_list = []
+$rentals = []
+
+def menu
+  puts 'Please choose an option by enterin a number:'
```

Comment:
- [ ] OPTIONAL
This intro text should read  `Please choose an option by entering a number` instead of `Please choose an option by enterin a number`. Kindly note the typo **enterin** and  ensure to improve upon this aspect